src/foam/u2/view/RadioView.js

```diff
@@ -20,10 +20,27 @@ foam.CLASS({
   name: 'RadioView',
   extends: 'foam.u2.view.ChoiceView',
 
+  axioms: [
+    foam.u2.CSS.create({
+      code: function CSS() {/*
+        ^{ }
+
+        ^ label {
+          position: relative;
+        }
+
+        .foam-u2-RadioContainer { }
```

Comment:
These are really generic names why are we defining them here?  What if someone adds a foam-u2-RadioContainer class or a foam.u2.Label ?